Assets/Scripts/Models/User.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using Enums;
+
+namespace Models
+{
+    public class User
```

Comment:
Don't we also want to store username and points/score?

Reply:
We should have some kind of tribe blueprint with the name, score card and starting inventory.